src/main/java/com/disi/TravelPoints/controller/VisitController.java

```diff
@@ -26,4 +29,13 @@ public ResponseEntity<Long> addVisit(@RequestBody AddVisitRequest request) throw
                     .build();
         }
     }
+
+    @GetMapping("/day-frequency")
+    public ResponseEntity<List<HourFrequencyVisitDTO>> getMonthFrequencyPerYear(
```

Comment:
I think the method name should be "getHourFrequencyPerDay" or something similar.